snmp/datadog_checks/snmp/data/default_profiles/aruba-clearpass.yaml

```diff
@@ -0,0 +1,224 @@
+extends:
+  - aruba.yaml
+
+sysobjectid:
+  - 1.3.6.1.4.1.14823.1.6.1
+metrics:
+  - MIB: HOST-RESOURCES-MIB
+    symbol:
+      name: cpu.usage
+      OID: 1.3.6.1.2.1.25.3.3.1.2.196608
+  - MIB: UCD-SNMP-MIB
+    symbol:
+      name: memory.total
+      OID: 1.3.6.1.4.1.2021.4.5.0
+  - MIB: UCD-SNMP-MIB
+    symbol:
+      name: memory.free
+      OID: 1.3.6.1.4.1.2021.4.6.0
+  - MIB: CPPM-MIB
+    table:
+      name: cppmSystemTable
+      OID: 1.3.6.1.4.1.14823.1.6.1.1.1.1
+    symbols:
+      - name: cppmSystemMemoryTotal
+        OID: 1.3.6.1.4.1.14823.1.6.1.1.1.1.1.12
+      - name: cppmSystemMemoryFree
+        OID: 1.3.6.1.4.1.14823.1.6.1.1.1.1.1.13
+      - name: cppmSystemDiskSpaceTotal
+        OID: 1.3.6.1.4.1.14823.1.6.1.1.1.1.1.14
+      - name: cppmSystemDiskSpaceFree
+        OID: 1.3.6.1.4.1.14823.1.6.1.1.1.1.1.15
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.14823.1.6.1.1.1.1.1.4
+          name: cppmSystemHostname
+        tag: cppm_system_hostname
+  - MIB: CPPM-MIB
+    table:
+      name: radiusServerTable
+      OID: 1.3.6.1.4.1.14823.1.6.1.1.2.1
+    symbols:
+      - name: radPolicyEvalTime
+        OID: 1.3.6.1.4.1.14823.1.6.1.1.2.1.1.1
+      - name: radAuthRequestTime
+        OID: 1.3.6.1.4.1.14823.1.6.1.1.2.1.1.2
+      - name: radServerCounterSuccess
+        OID: 1.3.6.1.4.1.14823.1.6.1.1.2.1.1.3
+      - name: radServerCounterFailure
+        OID: 1.3.6.1.4.1.14823.1.6.1.1.2.1.1.4
+      - name: radServerCounterCount
+        OID: 1.3.6.1.4.1.14823.1.6.1.1.2.1.1.5
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.14823.1.6.1.1.1.1.1.4
+          name: cppmSystemHostname
```

Comment:
Q: Is it possible to have two table rows with the same `cppmSystemHostname`?

Reply:
this table lists CPPM RADIUS server attributes and it is indexed by the `cppmSystemHostname` so it should be unique per entry